netsuite-openair/2.0/modules/ROOT/pages/netsuite-openair-connector-examples.adoc

```diff
@@ -0,0 +1,642 @@
+= NetSuite OpenAir 2.0 Examples - Mule 4
+
+This example shows how to add, modify, read, and delete two customers by using the same request. This example contains five flows:
```

Comment:
Suggestion: Add where we are adding and deleting users. I'm assuming it's a database (the NetSuite database?), so:
This example shows how to add, modify, read, and delete two customers from the NetSuite database by using the same request.